docker/python-verify/installed.Dockerfile

```diff
@@ -23,8 +23,8 @@ ARG MINIFI_VERSION
 ARG ARCHIVE_LOCATION
 ARG INSTALL_PACKAGE_CMD
 
-ARG UID=1000
-ARG GID=1000
+ARG UID=1001
+ARG GID=1001
```

Comment:
why change this?

Reply:
It already in use on ubuntu 24.04

Reply:
We could use that user, and skip adding a new one, if this only needs to work for ubuntu

Reply:
Im not sure if I follow.
On ubuntu 24.04 there is already a user called ubuntu with the uid 1000.
We use this dockerfile for all the distros we test on, and on the rest of them there is no user with uid 1000, so changing this arbitrary number seemed the easiest solution.
Moreover its probably more clean to use our own user with clear permissions.